tests/dummy/app/templates/list-controls.hbs

```diff
@@ -28,7 +28,7 @@
         {{#paper-subheader class="md-no-sticky"}}Clickable Items with Secondary Controls{{/paper-subheader}}
 
         {{#paper-item action="transitionToWifiMenu"}}
-          {{paper-icon "network-wifi"}}
```

Comment:
I think something changed but not sure where, you can also try here:
- http://jsbin.com/rifotulaja/edit?html,css,output
Change the underscores to dashes and it will break the icon,
ref. https://github.com/google/material-design-icons/issues/177